src/app/config/ethStrategies.ts

```diff
@@ -0,0 +1,2791 @@
+import { AssetType, Strategy } from "../types";
+
+import {
+  POINTS_ID_ETHENA_SATS_S3,
+  POINTS_ID_ETHENA_SATS_S4,
+  POINTS_ID_KARAK_S2,
+  POINTS_ID_SYMBIOTIC_S1,
+  POINTS_ID_MELLOW_S1,
+  POINTS_ID_MERITS_S1,
+  POINTS_ID_ZIRCUIT_S3,
+  POINTS_ID_ETHERFI_S4,
+  POINTS_ID_ETHERFI_S5,
+  POINTS_ID_VEDA_S1,
+  POINTS_ID_LOMBARD_LUX_S1,
+  POINTS_ID_RESOLV_S1,
+} from "./constants";
+
+export const ETH_CONFIG: Strategy[] = [
+  {
+    strategy: "Fluid: Smart Vault GHO-SUSDE/GHO-USDC Looping",
+    start: "Jun-05-2025 10:30:59 PM UTC",
+    owner: "0xd0e2eeeaf60908a9d53f9740533ce228866b384e",
+    fixedValue: { value: 21.06, asset: "USD" }, // leveraged value of collateral, not just USDe
+    points: [
+      {
+        type: POINTS_ID_ETHENA_SATS_S4,
+        expectedPointsPerDay: { value: 30, baseAsset: "USD" },
+      },
+    ],
+    externalAppURL: "https://fluid.io/vaults/1/125",
+  },
+  {
+    strategy: "Fluid: Smart Vault SUSDE-USDT/USDC-USDT Looping",
+    start: "Jun-05-2025 10:37:23 PM UTC",
+    owner: "0xb7385d98952ffc37788ba540959ebabf4d0975ae",
+    fixedValue: { value: 21.1, asset: "USD" }, // leveraged value of collateral, not just USDe
+    points: [
+      {
+        type: POINTS_ID_ETHENA_SATS_S4,
+        expectedPointsPerDay: { value: 30, baseAsset: "USD" },
+      },
+    ],
+    externalAppURL: "https://fluid.io/vaults/1/126",
+  },
+  {
+    strategy: "Fluid: Smart Vault USDE-USDT/USDC-USDT Looping",
+    start: "Jun-05-2025 10:41:47 PM UTC",
+    owner: "0xd564b1638c64e03c5596aa74c8907e56575c61ab",
+    fixedValue: { value: 25.03, asset: "USD" }, // leveraged value of collateral of LP, not just USDe
+    points: [
+      {
+        type: POINTS_ID_ETHENA_SATS_S4,
+        expectedPointsPerDay: { value: 30, baseAsset: "USD" },
+      },
+    ],
+    externalAppURL: "https://fluid.io/vaults/1/127",
+  },
+  {
+    strategy: "Pendle: Hold eUSDe YTs (13 Aug 2025)",
+    start: "Jun-05-2025 10:44:47 PM UTC",
+    owner: "0x22599909eb96dd5d7854261ea3f296ad93097013",
+    fixedValue: { value: 333.01, asset: "USD" }, // $5 of yts = 333.01 YTs x 50
+    points: [
+      {
+        type: POINTS_ID_ETHENA_SATS_S4,
+        expectedPointsPerDay: { value: 50, baseAsset: "USD" },
+      },
+    ],
+    externalAppURL:
+      "https://app.pendle.finance/trade/markets/0xe93b4a93e80bd3065b290394264af5d82422ee70/swap",
+  },
+];
```

Comment:
Small updates to use full leveraged amount for Fluid positions and use yt balance instead of USD value